ble2mqtt/devices/govee.py

```diff
@@ -0,0 +1,69 @@
+import logging
+from dataclasses import dataclass
+
+from bleak.backends.device import BLEDevice
+
+from ..devices.base import Sensor, SENSOR_DOMAIN, SubscribeAndSetDataMixin
+from ..protocols.govee import GoveeDecoder
+from ..utils import format_binary
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class SensorState:
+    battery: int = 0
+    temperature: float = 0
+    humidity: float = 0
+
+
+class GoveeTemperature(SubscribeAndSetDataMixin, Sensor):
```

Comment:
It looks like SubscribeAndSetDataMixin is not needed here as the device doesn't support active connections.
Can you please check that it works without this mixin?

Reply:
Hmm maybe RuuviTag doesn't need this either then and perhaps provided a bad example...

Reply:
Fixed